ocamltest/main.ml

```diff
@@ -235,6 +235,7 @@ let test_file test_filename =
        let make = try Sys.getenv "MAKE" with Not_found -> "make" in
        let initial_environment = Environments.from_bindings
            [
+             Builtin_variables.dev_null, "/dev/null";
```

Comment:
On Windows my understanding is that `/dev/null` is not available but `NUL` works as a null sink. (You only use this variable in tests that are disabled on windows, but it is available more widely so maybe we should try to get it right.)

Reply:
Honestly, my focus is on improving native Unix debugger and I have no plans to work on debugging for Windows in the near future. Plus I don't have access to an environment to test Windows changes right or really know what I'm doing on Windows in general.

IF there is a simple change to get Windows support here I can apply it. Otherwise I would ask to leave Windows support as a future task.